scapy/plist.py

```diff
@@ -299,10 +306,15 @@ def conversations(self, getsrcdst=None,**kargs):
             except:
                 #XXX warning()
```

Comment:
I think that we can remove it. Replacing it with a warning message might be better.